src/app.ts

```diff
@@ -179,13 +187,25 @@ export class App {
     };
 
     this.world.scene = sceneEl.object3D;
+    const scene = sceneEl.object3D;
+
+    // We manually call scene.updateMatrixWolrd in mainTick
+    scene.autoUpdate = false;
```

Comment:
Is this manual updating for post processing? Can auto update break the post processing?

Reply:
Post processing can call render multiple times, so it makes more sense to just manually update ourselves once. This is also something we wanted to do anyway so that we can easily see where in the frame matrix updates are happening and schedule systems before/after that.

Reply:
> Post processing can call render multiple times, so it makes more sense to just manually update ourselves once.

OK, makese sense to me. I thought `postprocessing` lib set `scene.autoUpdate` `false` but if it doesn't, setting `scene.autoUpdate` `false` and manually calling update matrix world in our code makes sense.